### PR TITLE
Enable web integration tests in CI

### PR DESCRIPTION
**Pull request by stuartmorgan**
- Adds web support to the drive-examples script comment
- Enables driving the web integration tests on master
- Moves web integration tests to the main Dockerfile image since recent Chrome support has been added to the driver

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [x] I signed the [CLA].
- [ ] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [ ] I updated CHANGELOG.md to add a description of the change.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test exempt.
- [x] All existing and new tests are passing.

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[C++, Objective-C, Java style guides]: https://github.com/flutter/engine/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning

### CODE REVIEW

File: script/tool/lib/src/drive_examples_command.dart [hunk 1]
```diff
@@ -180,30 +190,35 @@ Tried searching for the following:
...
-      return (isAndroidPlugin(plugin, fileSystem));
-    }
     // When we are here, no flags are specified. Only return true if the plugin
     // supports Android for legacy command support. TODO(cyanglaz): Make Android
     // flag also required like other platforms (breaking change).
     // https://github.com/flutter/flutter/issues/58285
     return isAndroidPlugin(plugin, fileSystem);
   }
 }
+//  - flutter drive -v --driver=test_driver/integration_test.dart --target=integration_test/example_test.dart
```
Comment (ditman) [inline script/tool/lib/src/drive_examples_command.dart:224]: Remove?

**Review by ditman — APPROVED** [review]
This doesn't seem to be running all the tests that are available? I was probably not self-consistent with naming files? :/

File: script/tool/lib/src/drive_examples_command.dart (hunk 1 above)
Comment (stuartmorgan) [reply script/tool/lib/src/drive_examples_command.dart:224]: Oops. Can you tell I needed an extra paste buffer? 😬

**Comment by stuartmorgan** [comment]
> This doesn't seem to be running all the tests that are available? I was probably not self-consistent with naming files? :/

Capturing from Discord for posterity: the script requires that the driver file end in `_test.dart`; many web examples use `integration_driver.dart` instead. That will be fixed separately.

(I think there's a bug in the script where that failure skips more code than its supposed to, as there's logging later on about not being able to find a driver that's not reached.)

**Pull request merged**